### DISCUSSION TITLE
What happens to the debugger stuff if a user does not ...

### DISCUSSION THREAD

**Discussion by Shluffy** [ScratchAddons/ScratchAddons#2791] [Help] [answered]
... have scratch addons enabled, or that addon enabled? or is it only for the creator?

**Comment by TheColaber** [comment, marked as answer]
the blocks become your average scratch custom blocks. they won't break the project in any way, but it is recommended to remove the blocks before you share a project.

**Reply by Shluffy** [reply]
will they work for another sa user with the addon enabled?

**Reply by mxmou** [reply]
@Shluffy They will.
